docs/Configuration.rst

```diff
@@ -2908,46 +2908,44 @@ These Portal caches can be overridden in
 ::
 
     # Portal Cache Configuration - MOP session Manager
-    #  - Standalone (live time in seconds)
-    exo.cache.portal.mop.maxsize=5000
-    exo.cache.portal.mop.livetime=600
-    #  - Cluster (expiration in milliseconds)
-    exo.cache.portal.mop.maxnodes=5000
-    exo.cache.portal.mop.expiration=600000
+    exo.cache.portal.mop.MaxNodes=1400
+    exo.cache.portal.mop.TimeToLive=86400
+    exo.cache.portal.mop.strategy=LIRS
+    # For Cluster mode
+
+    exo.cache.portal.mop.cacheMode=replication
 
     # Portal Cache Configuration - Navigation Service
-    #  - Standalone (live time in seconds)
-    exo.cache.portal.navigation.maxsize=5000
-    exo.cache.portal.navigation.livetime=600
-    #  - Cluster (expiration in milliseconds)
-    exo.cache.portal.navigation.maxnodes=5000
-    exo.cache.portal.navigation.expiration=600000
+    exo.cache.portal.navigation.MaxNodes=700
```

Comment:
You used the values defined in exo-sample.properties right ? Shouldn't we use the default value instead (defined in https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties) ? We need these default values somewhere IMO.

Reply:
Yes i used values of exo-sample.properties file OK i will change them

Reply:
Values updated